smelli/classes.py

```diff
@@ -135,6 +137,58 @@ def log_likelihood_sm(self):
             self._log_likelihood_sm = self._log_likelihood(flavio.WilsonCoefficients())
         return self._log_likelihood_sm
 
+    @property
+    def obstable_sm(self):
+        if self._obstable_sm is None:
+            info = tree()  # nested dict
+            for flh_name, flh in self.fast_likelihoods.items():
+                # loop over fast likelihoods: they only have a single "measurement"
+                m = flh.pseudo_measurement
+                ml = flh.full_measurement_likelihood
+                pred_sm = ml.get_predictions_par(self.par_dict,
+                                                flavio.WilsonCoefficients())
+                sm_cov = flh.sm_covariance.get(force=False)
+                _, exp_cov = flh.exp_covariance.get(force=False)
+                inspire_dict = self._get_inspire_dict(flh.observables, ml)
+                for i, obs in enumerate(flh.observables):
+                    info[obs]['lh_name'] = flh_name
+                    info[obs]['name'] = obs if isinstance(obs, str) else obs[0]
+                    info[obs]['theory_sm'] = pred_sm[obs]
+                    info[obs]['th. unc.'] = np.sqrt(sm_cov[i, i])
+                    info[obs]['experiment'] = m.get_central(obs)
+                    info[obs]['exp. unc.'] = np.sqrt(exp_cov[i, i])
+                    info[obs]['exp. PDF'] = NormalDistribution(m.get_central(obs), np.sqrt(exp_cov[i, i]))
+                    info[obs]['inspire'] = sorted(set(inspire_dict[obs]))
+                    info[obs]['ll_sm'] = m.get_logprobability_single(obs, pred_sm[obs])
+                    info[obs]['ll_central'] = m.get_logprobability_single(obs, m.get_central(obs))
```

Comment:
Here the column names are not uniform. The "old" ones are abbreviations, the new ones written like variables. So I would replace `theory_sm` simply by `SM`. `ll_sm`, `ll_central` and `lh_name` are deleted later, so it doesn't really matter.

Reply:
At the moment, `theory_sm` is also deleted later. Do you think it should be included in the obstable output?

Reply:
Oh. Well it's not strictly necessary, I thought it was intentional. Can also be omitted.

Reply:
I just realized that `theory_sm` it is not used at all later on. So I will just completely remove it.

Reply:
Done in https://github.com/smelli/smelli/pull/1/commits/3a768107a4709b27a728ce160e32eb3e7b794920.